core/src/main/scala/com/yahoo/maha/core/BaseUTCTimeProvider.scala

```diff
@@ -286,9 +288,13 @@ class BaseUTCTimeProvider extends UTCTimeProvider with Logging {
 
   def validateFilters(dayFilter: Filter, hourFilter: Option[Filter], minuteFilter: Option[Filter]): Boolean = {
     if (!hourFilter.isDefined && !minuteFilter.isDefined) return true
+    info(s"Hour or minute filter is not defined")
     if (!hourFilter.isDefined && minuteFilter.isDefined) return false
+    info(s"validated minuteFilter is not defined without hourFilter")
     if (hourFilter.isDefined && dayFilter.operator != hourFilter.get.operator) return false
+    info(s"hourFilter is defined both day and hour filter operator are the same")
     if (hourFilter.isDefined && minuteFilter.isDefined && hourFilter.get.operator != minuteFilter.get.operator) return false
+    info(s"minute filter operator is same as hour filter operator")
```

Comment:
These logs should be clubbed.  Maybe make a secondary single log which gives you all of those values ahead of their checks.